daprdocs/content/en/reference/annotations/annotations-overview.md

```diff
@@ -0,0 +1,55 @@
+---
+type: docs
+title: "Dapr annotations spec"
```

Comment:
This page actually compares annotations, CLI arguments and daprd arguments so I would not call it "annotations spec" (Also file name and location should not be "annotations") since annotations are only in the k8s space. I think this can be put directly under reference (no additional folder) and be called "Dapr arguments and annotations"